src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobMetadataStore.cs

```diff
@@ -92,6 +97,12 @@ public class BlobMetadataStore : IMetadataStore
         }
         catch (Exception ex)
         {
+            if (ex is NotSupportedException)
+            {
+                _telemetryClient
+                    .GetMetric("JsonSerializationException", "ExceptionType")
```

Comment:
Both of these should probably be constants, and I'd suggest just `Type` for the dimension name.